### DISCUSSION TITLE
SOFT FAIL - I can't send emails

### DISCUSSION THREAD

**Discussion by kkeetto** [postalserver/postal#1761] [Help with using Postal]
Hello, hope u are doing good. I'm new to this, I recently installed postal on a digitalocean vps, all went well, faced some difficulties but I was able to get it done. Now the problem is when i try to send to gmail or hotmail (and im assuming any mail box) it gives me this error after being queed for 5 mins :

![Inkedgithub 1_LI](https://user-images.githubusercontent.com/95940892/145640684-d2c2ecc0-4818-476b-b062-d43817dbb1e3.jpg)
 
on the postal logs i get this :

![github 2](https://user-images.githubusercontent.com/95940892/145640950-dafafbef-b5c7-498f-9425-cb0b0da0e57b.PNG)

my postal status :
![2525](https://user-images.githubusercontent.com/95940892/145641084-85416634-c9c0-49ff-9a11-edb52fc4727d.PNG)

important note i guess: I didn't use the domain that contains postal as a sending domain ! I linked a second domain to postal for sending ( we will call the domain I use to access my smtp : ''domain A''   /   and the domain that I linked to postal for sending : ''domain B") ( I thought it's possible bcs I figured that you can add multiple domains ) so maybe that's the problem 

Now im going to show u my postal.yml file : 

general:
   This can be changed to allow messages to be sent from multiple IP addresses
  use_ip_pools: false

web:
   The host that the management interface will be available on
  host: server._domain A_
   The protocol that requests to the management interface should happen on
  protocol: https

web_server:
   Specify configuration for the Postal web server
  bind_address: 127.0.0.1
  port: 5000

smtp_server:
   Specify configuration to the Postal SMTP server
  port: 25

logging:
   Specify options for the logging
  stdout: true

main_db:
   Specify the connection details for your MySQL database
  host: 127.0.0.1
  username: root
  password: postal
  database: postal

message_db:
   Specify the connection details for your MySQL server that will be house the
   message databases for mail servers.
  host: 127.0.0.1
  username: root
  password: postal
  prefix: postal

rabbitmq:
 - Specify connection details for your RabbitMQ server
  host: 127.0.0.1
  username: postal
  password: postal
  vhost: postal

dns:
   Specify the DNS records that you have configured. Refer to the documentation at
   https://github.com/atech/postal/wiki/Domains-&-DNS-Configuration for further
   information about these.
  mx_records:
    - mx.postal.DomainB
  smtp_server_hostname: postal.**DomainB**
  spf_include: spf.postal.**DomainB**
  return_path: rp.postal.**DomainB**
  route_domain: routes.postal.**DomainB**
  track_domain: track.postal.**DomainB**
-smtp :
 Specify an SMTP server that can be used to send messages from the Postal management
 system to users. You can configure this to use a Postal mail server once the
  your installation has been set up.
  host: 127.0.0.1
  port: 2525
  username:  Complete when Postal is running and you can
  password:  generate the credentials within the interface.
  from_name: Postal
  from_address: server.**DomainB**

Ps: Please bear with me i don't know nothing about coding, im all ears thank you !

**Comment by willpower232** [comment]
It looks like you might need to ask DigitalOcean if they are blocking your email ports. If they aren't then you could try mxtoolbox.com to see if there are any other issues.

**Reply by kkeetto** [reply]
Thank you for answering 
I have sent a ticket to DigitalOcean, by email ports u mean port 25 ? I have checked that port 25 is working fine :
![githubbb](https://user-images.githubusercontent.com/95940892/145685054-3b1c3bc9-311a-4f5e-8b55-7785e538eb92.PNG)
another question is what should i check on mxtoolbox.com ? i mean which domain ? and check for what ? 
Thank you again for your response !

**Reply by willpower232** [reply]
The port may be working on your server but its possible that Digital Ocean have blocked that port for your account so that you cannot connect to other port 25s around the internet.

Have a look at the "supertool", if you give it the IP or domain name of your Postal server then it can do all of the tests.
